src/main/java/entities/User.java

```diff
@@ -97,12 +98,18 @@ public void addCart() {
     public void removeCart() {
 
     }
-    public void writeReview() {
+    public void writeReview(User reviewed, int rating) {
+        new ReviewCreator().createReview(this, reviewed, rating);
 
     }
     public void removeReview() {
 
     }
+
+    //TODO: check if this is necessary/who gets to hold reviews
+    public void addReview(Review review){
+        this.reviews.add(review);
+    }
 //    public int calculateRating() {
```

Comment:
loop over reviews and calculate average? will need getters for this

Reply:
just added the calculateRating feature, is that what you meant?